src/Suave/Types.fs

```diff
@@ -355,7 +355,7 @@ type Host =
   /// The client's Host header is this value
   | ClientOnly of string
   /// The
-  | Forwarded of forwarded_for:string * Host
+  | Forwarded of string * Host
```

Comment:
I get these errors if I revert it: "Unexpected symbol ':' in member definition" and "The type 'forwarded_for' is not defined". What do you suggest I do?